### ISSUE TITLE
Firejail versions 0.9.54~rc1_1 & 0.9.54~rc2_1 are causing Mozilla Thunderbird to crash

### ISSUE THREAD

**Issue by SuperSentient** [netblue30/firejail#1946]
Theres a problem with Firejail versions 0.9.54~rc1_1 & 0.9.54~rc2_1. There causing Mozilla Thunderbird to crash. Thunderbird was working just fine before FJ versions 0.9.54~rc1_1 & 0.9.54~rc2_1 when I run Thunderbird without Firejail it works just fine. Please fix this.

**Comment by netblue30** [comment]
I cannot reproduce it on Debian stable. What distribution are you using? Also, how do you start the sandbox?

**Comment by SuperSentient** [comment]
Linux Mint 17.2. I start it with the command firejail thunderbird from the icon settings.

**Comment by netblue30** [comment]
I'll give it a try!

**Comment by SuperSentient** [comment]
@ chiraag-nataraj
Yes its still an issue even with version 0.9.54.

**Comment by chiraag-nataraj** [comment]
What happens if you use `firejail --noprofile thunderbird`?

**Comment by SuperSentient** [comment]
@  **chiraag-nataraj** 

Thanks thats seems to work. But isn't that the same as not using Firejail?

**Comment by chiraag-nataraj** [comment]
@SuperSentient Yes. Now, can you go through, comment out all of the lines in the profile, and slowly uncomment them until you find what breaks the profile?

**Comment by SuperSentient** [comment]
@**chiraag-nataraj** Thanks but I'm not a programmer I joined GitHub so that I can report bugs and possibly try out new software. Anyway heres the all the lines from the terminal when I run **firejail --noprofile thunderbird** : Parent pid 9162, child pid 9163
Child process initialized in 13.13 ms

(thunderbird:2): GLib-GObject-CRITICAL **: g_object_unref: assertion 'object->ref_count > 0' failed

(thunderbird:2): GLib-GObject-CRITICAL **: g_object_unref: assertion 'object->ref_count > 0' failed

I cant make sense out of it.

**Comment by SuperSentient** [comment]
@ **Fred-Barclay** 
What do you mean by #? Also I don't notice any problems with Thunderbird in Kubuntu 18.04 which I'm planning to replace my Linux Mint KDE 17.2 with latter this year.

**Comment by SuperSentient** [comment]
So I should try to enter **# include /etc/firejail/firefox.profile** in the terminal?

**Comment by chiraag-nataraj** [comment]
No.
Terminology:
profile: the file(s) read by `firejail` in order to sandbox the program. Files installed by `firejail` will be found in `/etc/firejail/` or `/usr/local/etc/firejail`, while custom profiles that you develop should be placed in `~/.config/firejail`. These files will usually end with `.profile`, although many common files have been refactored and end with `.inc`.
comment: to place a `#` sign at the beginning of the specified line. This should be done _within the profile file_.
uncomment: to remove the `#` you placed at the beginning of the line. Again, this should be done _within the profile file_.

Does that help?

**Comment by Fred-Barclay** [comment]
@SuperSentient 
This might be the easiest way...
Please copy the following into the file /etc/firejail/thunderbird.profile (you'll need to edit it as root):
```
#include /etc/firejail/thunderbird.local
#include /etc/firejail/globals.local
#noblacklist ${HOME}/.cache/thunderbird
#noblacklist ${HOME}/.gnupg
#noblacklist ${HOME}/.thunderbird
#mkdir ${HOME}/.cache/thunderbird
#mkdir ${HOME}/.gnupg
#mkdir ${HOME}/.thunderbird
#whitelist ${HOME}/.cache/thunderbird
#whitelist ${HOME}/.gnupg
#whitelist ${HOME}/.thunderbird
#ignore private-tmp
#read-only ${HOME}/.config/mimeapps.list
#writable-run-user
#include /etc/firejail/firefox.profile
```

Save the file.
Then remove the `#` from the first line (so it will look like `include /etc/firejail/thunderbird.local`). Save the file. Run `firejail thunderbird`.
Does it work? If so, now remove the `#` from the next line (it will now look like `#include /etc/firejail/globals.local`). Save the file. Run `firejail thunderbird`.

Repeat this process until `firejail thunderbird` no longer works. Once that occurs, re-add the `#` to the beginning of the line, save the file, and move on the next line.
Once you're on the next line, remove the `#` from it. Save the file. Run `firejail thunderbird`.

Continue on until you've done all the lines. Then let us know which lines you had to re-add the `#` to for `firejail thunderbird` to work.

Hope this helps!
Fred

**Comment by chiraag-nataraj** [comment]
@SuperSentient Were you able to figure this out?
